README.rst

```diff
@@ -29,57 +31,89 @@ Further documentation is available on
 https://invenio-curations.readthedocs.io/
 
 
+Update ``invenio.cfg``
+----------------------
 
-## Update invenio.cfg
-### Add notification builders and entity resolver for groups
+Add `notification builders` and `entity resolver` for `groups`
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-from invenio_curations.notifications.builders import (
-    CurationRequestAcceptNotificationBuilder,
-    CurationRequestSubmitNotificationBuilder,
-)
+Currently, requests can only be sent to a single `receiver`.
+However, curation reviews are typically performed by a `group` of people rather than one single fixed `user`.
+Thus, the curation requests are sent to a `group` rather than a single `user` in the system so that all users with a certain `role` can receive and act on curation requests.
 
-from invenio_records_resources.references.entity_resolvers import ServiceResultResolver
+To enable resolution of user groups, an `entity resolver` has to be added to the configuration.
```

Comment:
this paragraph is based on educated guesswork, is it even true?

Reply:
That sounds correct. Setting the group as receiver allows to access the request. For the record itself, there is an additional access granted to the group on request creation.